packages/compiler/src/render3/view/template.ts

```diff
@@ -1083,23 +1092,17 @@ export class TemplateDefinitionBuilder implements t.Visitor<void>, LocalResolver
   }
 
   private updateInstructionChain(
-      nodeIndex: number, span: ParseSourceSpan|null, reference: o.ExternalReference,
-      callsOrFn?: o.Expression[][]|(() => o.Expression[][])) {
+      nodeIndex: number, reference: o.ExternalReference,
+      propertyBindings: ChainableBindingInstruction[]) {
```

Comment:
Nit: Since this can receive attribute bindings, perhaps we should generalize the name?

```suggestion
      bindings: ChainableBindingInstruction[]) {
```